Server/src/controllers/authenticationController.ts

```diff
@@ -64,6 +65,44 @@ export class AuthenticationController {
         }
     }
 
+    private validateUserDetailRequest(request: Request): boolean {
+        if (request.query.hasOwnProperty('password')) {
```

Comment:
Do we need to check organization here? Or just password?

Reply:
i think just checking for password is fine. Since we already checked for email, if there is no password , it won't make sense to have a valid request. If needed i can add the check for the organization name .

Reply:
What happens if a user is only passing an update to organization without a password update?

Reply:
uhh, if i check off the password field and only input email and organization name, i get an invalid request. however, it works if i leave the password field checked but empty

Reply:
fixed on the latest commit. we can now update the organization name without the password update